### PR TITLE
Fix early game over in soviet07

### PR DESCRIPTION
**Pull request by ghost**
- Fix https://github.com/OpenRA/OpenRA/issues/9561
- Add a missing healcrate

### CODE REVIEW

File: mods/ra/maps/soviet-07/soviet07.lua [hunk 1]
```diff
@@ -239,7 +239,9 @@ IntroSequence = function()
 		timerStarted = true
 	end)
 	Trigger.OnAllKilled(StartingUnits, function()
```
Comment (abcdefg30) [inline mods/ra/maps/soviet-07/soviet07.lua]: Couldn't we just drop this entirely?
As you have a `player.HasNoRequiredUnits()` check in `Tick`.

Comment (Biofreak1987) [inline mods/ra/maps/soviet-07/soviet07.lua]: The point of it was that you need a human unit to interact with the security system and can't use an animal for that job. I'll remove that Trigger if the majority wants it gone.

**Comment by abcdefg30** [comment]
:+1:

File: mods/ra/maps/soviet-07/soviet07.lua (hunk 1 above)
Comment (abcdefg30) [inline mods/ra/maps/soviet-07/soviet07.lua]: Didn't think about that, good point.

Comment (pchote) [inline mods/ra/maps/soviet-07/soviet07.lua]: Please add a comment to the code explaining this.

**Comment by Biofreak1987** [comment]
Added a comment for the early game over and an OpenRA.sln entry for the lua script as pointed out by clemty

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/8be49ecdf28b5ee2d1a6b27de4980129561153d5)

**Pull request merged**